modules/oxide/src/less/theme/components/inline-toolbar/inline-toolbar.less

```diff
@@ -0,0 +1,26 @@
+//
+// Inline Toolbar
+//
+.tox {
+  .tox-inline-toolbar {
+    // CSS variable for gap between anchor and toolbar
+    --inline-toolbar-gap: 6px;
+
+    // CSS Anchor Positioning (required for anchor() to work)
+    position: absolute;
+
+    // Enable automatic flip when toolbar would overflow viewport, very new, not recongnized by stylelint yet
+    // stylelint-disable-next-line property-no-unknown -- CSS Anchor Positioning spec (Chrome 125+)
+    position-try-fallbacks: flip-block, flip-inline, flip-block flip-inline;
+    z-index: 9999;
```

Comment:
Is there a smaller number we could use? I'm imagining we could support dropdowns from this toolbar at some point, where they would need to be above this.